htmresearch/frameworks/nlp/classify_document_fingerprint.py

```diff
@@ -163,7 +162,7 @@ def inferToken(self, token, reset=0, returnDetailedResults=False,
       document = " ".join(self.currentDocument)
       sensor = self.sensorRegion.getSelf()
       sensor.addDataToQueue(token=document, categoryList=[None],
-                            sequenceId=-1, reset=0)
+                            sequenceId=-1, reset=reset)
```

Comment:
It would be nice if `reset` had a more descriptive name.  It's not well documented in the docstring, and it comes up in various places (like this line) and it's not clear *what* `reset` is supposed to do.  Also, since it's being used as a boolean, it would be either `True` or `False` and not integer `1` or `0`.